.github/workflows/deploy-server.yml

```
@@ -0,0 +1,33 @@
+name: deploy-server
+on:
+  workflow_run:
+    workflows: ["lint-test-build"]
+    branches: [production]
+    types: [completed]
+
+env:
+  server_dir: mobile-browser-based-version/server
+  server_url: https://deai-313515.ew.r.appspot.com/
+
+jobs:
+  deploy:
+    name: deploy-gae
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ env.server_dir }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: deploy
+        uses: google-github-actions/deploy-appengine@v0.2.0
```

Is there a reason behind using an old version?

```suggestion
        uses: google-github-actions/deploy-appengine@v0.8.0
```

btw, to ensure that it works, I usually force it to run, by branching of and changing the trigger to `on: push`

---

thanks for upgrading it, the reason for old version: copy pasted code. How to test: thanks that's a good tip, will test it out.